form-flow-valtimo/src/main/kotlin/com/ritense/valtimo/formflow/mapper/FormFlowProcessLinkMapper.kt

```diff
@@ -76,7 +75,7 @@ class FormFlowProcessLinkMapper(
             throw RuntimeException("FormFlow definition not found with id ${createRequestDto.formFlowDefinitionId}")
         }
         return FormFlowProcessLink(
-            id = UUID.randomUUID(),
+            id = createRequestDto.processLinkId,
```

Comment:
Because we can now provide an id from the outside, I would expect a check that it is indeed a new processlink and not an existing one.

Reply:
It is, but there are separate method for updating and creating process links in the `ProcessLinkService`. We don't want it to be possible to update an existing one using the `createProcessLink` method. The JPA repository uses the `save` method in either case so wouldn't care about an existing entity.

Reply:
Will revert to having a return var instead of provided id.